tests/test_feed.py

```diff
@@ -1,4 +1,6 @@
 import datetime
+import pandas as pd
```

Comment:
Please group the pandas import with the pytest import according to these guidelines: https://www.python.org/dev/peps/pep-0008/#imports.